hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -258,6 +259,10 @@ public final class OzoneManagerProtocolClientSideTranslatorPB
       = new ThreadLocal<>();
     
   private boolean s3AuthCheck;
+
+  public static final int BLOCK_ALLOCATION_RETRY_COUNT = 5;
+  public static final int BLOCK_ALLOCATION_RETRY_WAIT_TIME_MS = 3000;
```

Comment:
1. Is 15 seconds enough for SCM to go out of safe mode during rolling restart?
2. Would this be made configurable later?

Reply:
Not exactly sure about the time and it may depend upon the environment load too.
Shall I add new config about retry count and also increase default wait to 30sec ?

Reply:
I'm fine with either way. We can make it configurable later if needed.